### ISSUE TITLE
pac canvas not available

### ISSUE THREAD

**Issue by anskanth** [MicrosoftDocs/powerapps-docs#2457]
Hi

There is no mention of how do we get this preview or the preview version of CLI. If I used `pac install latest` it is not giving me the Canvas actions. Would be great, if the docs can be updated on which version (public preview) this is available and how to get that version as well.

Thank You

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a0e3024-713b-e281-1417-21106fda3045
* Version Independent ID: 177d4ac9-7856-cbea-dc61-a7d11a909273
* Content: [Microsoft Power Platform CLI - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/powerapps-cli)
* Content Source: [powerapps-docs/developer/data-platform/powerapps-cli.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/powerapps-cli.md)
* Service: **powerapps**
* GitHub Login: @Nkrb
* Microsoft Alias: **nabuthuk**

**Comment by karankhosla12** [comment]
I guess this issue is resolved, i faced the same issue yesterday and it shows up now

**Comment by anskanth** [comment]
@karankhosla12 it didn't worked for me.. I just tried to uninstall fully and installed.. no luck.. 
![image](https://user-images.githubusercontent.com/20016296/122288847-f222c980-cee9-11eb-820a-ee4c0d5a53e7.png)

**Comment by brasov2de** [comment]
Hi @anskanth , 
Until it will be available, you could use the Power Platform VSCode Extension (https://marketplace.visualstudio.com/items?itemName=microsoft-IsvExpTools.powerplatform-vscode)
There you get the pac version 1.6.7. having the canvas options.

**Comment by anskanth** [comment]
Thanks @brasov2de . I saw that.. but my problem is, I'm trying to use this in an azure build pipelines and infact trying to follow your [blog ](https://dianabirkelbach.wordpress.com/2020/12/08/devops-dataverse-alm-using-power-apps-cli-a-source-centric-approach-part-2/) and tweak it slightly to get my canvas app extracted and commited in to Git. :).

Having it part of VSCode extension might not work for me. Surprised to see why the preview nuget package is not yet published. I looked in to the extension logic and apparently it is pulling the version from some private repository of MS, not sure why it has to be so private whilst it is part of extension.
